CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+## 1.1.1 - [#5](https://github.com/openfisca/extension-template/pull/5)
+
+* Technical improvement: adapt to version `17.0.0` of Openfisca-Core and version `1.2.4` of Country-Template
+* Details:
+  - Transform XML parameter files to YAML parameter files.
+  - Add a changelog
```

Comment:
When running the tests, they failed due to non Yaml files being present in the directory.
Might have been my configuration :)